OpenFeature.podspec

```diff
@@ -0,0 +1,20 @@
+Pod::Spec.new do |s|
+  s.name             = 'OpenFeature'
+  s.version          = '0.3.0'
```

Comment:
We'll probably need to update Release Please to manage this version automatically.

Reply:
the `update_podspec.sh` script should handle this in the release action.

Reply:
You shouldn't need the script. If you add `# x-release-please-version` after the version and update the [Release Please config](https://github.com/open-feature/swift-sdk/blob/main/release-please-config.json#L10) to manage this file it will manage the version automagically.